src/ace/ari_text/encode.py

```diff
@@ -53,12 +53,17 @@ def percent_encode(text):
 
 def encode_datetime(value):
```

Comment:
Do you mind if I type-annotate some of these functions to make more clear what they are doing?